content/pt-br/docs/reference/using-api/client-libraries.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Bibliotecas de clientes
+reviewers:
+- ahmetb
+content_type: conceito
+weight: 30
+---
+
+<!-- overview -->
+Esta página contém uma visão geral das bibliotecas clientes para usar a API Kubernetes de várias linguagens de programação.
+
+
+<!-- body -->
+Para desenvolver aplicações usando a [API REST do Kubernetes](/docs/reference/using-api/),
+você não precisa implementar as chamadas da API e os tipos de solicitação/resposta. 
+Você pode usar uma biblioteca cliente para a linguagem de programação que está usando.
+
+As bibliotecas clientes geralmente lidam com tarefas comuns, como autenticação para você.
```

Comment:
Maybe can be better: `As bibliotecas clientes geralmente lidam com tarefas comuns para você, como autenticação`